presto-native-execution/presto_cpp/presto_protocol/Base64Util.cpp

```diff
@@ -263,6 +263,9 @@ velox::VectorPtr readRleBlock(
   }
 
   if (type->kind() == velox::TypeKind::SHORT_DECIMAL ||
+      // Uncomment the following lines can fix the issue:
```

Comment:
This method is expected to read a single NULL value of type 'type'. Hence, I suggest to replace all code from L265 to the end of the function with

```
return velox::BaseVector::createNullConstant(type, positionCount, pool);
```